src/components/Thought.tsx

```diff
@@ -179,7 +179,16 @@ const ThoughtContainer = ({
     attributeEquals(state, head(rootedParentOf(state, parentOf(simplePath))), '=view', 'Table'),
   )
 
-  const hideBullet = useHideBullet({ children, env, hideBulletProp, isEditing, simplePath, thoughtId })
+  const isInContextView = useSelector(state => {
+    /** Check if the thought is in the context view. */
+    const checkContextView = (currentPath: Path): boolean => {
+      if (currentPath.length <= 1) return false
+      return isContextViewActive(state, parentOf(currentPath))
+    }
+    return checkContextView(path)
+  })
```

Comment:
This can. be simplified to:

```ts
const showContexts = useSelector(state => isContextViewActive(state, rootedParentOf(state, path)))
```

or even:

```ts
const showContexts = useSelector(state => isContextViewActive(state, parentOf(path)))
```

Reply:
I got it. I will regard that.